src/pudl/etl/__init__.py

```diff
@@ -64,6 +64,7 @@
 
 
 core_module_groups = {
+    "core_eia176": [pudl.transform.eia176],
```

Comment:
I went ahead and made this 1-character change in order to get this merged in before I leave.